src/util/classes.ts

```diff
@@ -20,26 +21,63 @@ export interface Category {
   children?: Category[];
 }
 
-const COLOR_UNCAT = '#CCC';
+// https://colorhunt.co/palette/4802
+const COLOR_GREEN = '#96cd39',
+  COLOR_SUPER_GREEN = '#54e346',
+  COLOR_UNCAT = '#CCC',
+  COLOR_ORANGE_GREEN = '#f5ff65',
```

Comment:
Maybe rename/change this slightly to a yellow?

Reply:
Fixed.